contracts/modules/dispute/DisputeModule.sol

```diff
@@ -341,6 +365,12 @@ contract DisputeModule is
         return $.disputeCounter;
     }
 
+    /// @notice Returns the arbitration policy cooldown
+    function arbitrationPolicyCooldown() external view returns (uint256) {
+        DisputeModuleStorage storage $ = _getDisputeModuleStorage();
+        return $.arbitrationPolicyCooldown;
```

Comment:
Directly return might save a little gas and code size.
`return _getDisputeModuleStorage().arbitrationPolicyCooldown`

Reply:
addressed in https://github.com/storyprotocol/protocol-core-v1/pull/281/commits/7ee122a7561bd398a573ba5bacbc00d5704ac712